waggle-dance-api/src/test/java/com/hotels/bdp/waggledance/api/model/PrimaryMetaStoreTest.java

```diff
@@ -119,4 +119,5 @@ public void constructorWithArrayListForWhtelist() {
     assertThat(store.getAccessControlType(), is(accessControlType));
     assertThat(store.getWritableDatabaseWhiteList(), is(whitelist));
   }
+
```

Comment:
There is a minor spelling mistake in the function name: `constructorWithArrayListForWhtelist` -> `constructorWithArrayListForWhitelist`

Reply:
Done :)